zkevm-circuits/src/evm_circuit/execution/jumpi.rs

```diff
@@ -74,6 +63,22 @@ impl<F: Field> ExecutionGadget<F> for JumpiGadget<F> {
         };
         let same_context = SameContextGadget::construct(cb, opcode, step_state_transition);
 
+        // Lookup opcode at destination when should_jump
+        cb.condition(should_jump.clone(), |cb| {
```

Comment:
Nit-pick: for consistency:
```suggestion
        cb.condition(should_jump.expr(), |cb| {
```